client/js/components/map/map/MapView.vue

```diff
@@ -240,15 +240,7 @@ export default {
     },
 
     center () {
-      if (this.procedureCoordinates) {
-        const array = this.procedureCoordinates.split(',')
-        return [
-          Number(array[0]),
-          Number(array[1])
-        ]
-      } else {
-        return false
-      }
+      return this.procedureCoordinates ? this.procedureCoordinates : false
```

Comment:
```suggestion
      return this.procedureCoordinates ?? false
```

Reply:
may this can be an empty array too? then wouldn't mix types

Reply:
If we return an empty array, we need to change code in DpOlMap and various other places I think. Wouldn't do this now, because of time. But I added the check for empty array here [2b5514e](https://github.com/demos-europe/demosplan-core/pull/3059/commits/2b5514e07618c138ec193957c1eb5a70c3102090)